pulp_ansible/app/viewsets.py

```diff
@@ -173,6 +174,10 @@ class Meta:
         fields = ["namespace", "name", "version", "q", "is_highest", "tags"]
 
 
+class CollectionRemoteFilter(RemoteFilter):
+    url = filters.CharFilter(field_name="url")
```

Comment:
I think you need to define a Meta class here. See instructions: https://github.com/pulp/pulpcore/blob/main/pulpcore/app/viewsets/repository.py#L213-L221

Reply:
And please try without this line. The filter should be generated from the fields attribute.